edgelet/Cargo.lock

```diff
@@ -1533,12 +1542,13 @@ dependencies = [
 
 [[package]]
 name = "native-tls"
-version = "0.2.1"
+version = "0.2.7"
```

Comment:
Updated the Cargo.toml to 0.2.7 and committed the lock file resulting in the update.

Reply:
Or just `cargo update -p native-tls`